packages/docusaurus-theme-openapi-docs/package.json

```diff
@@ -28,15 +28,15 @@
     "watch": "concurrently --names \"lib,lib-next,tsc\" --kill-others \"yarn babel:lib --watch\" \"yarn babel:lib-next --watch\" \"yarn tsc --watch\""
   },
   "devDependencies": {
-    "@docusaurus/types": "^3.5.0",
+    "@docusaurus/types": "3.5.2-canary-6122",
     "@types/crypto-js": "^4.1.0",
     "@types/file-saver": "^2.0.5",
     "@types/lodash": "^4.14.176",
     "concurrently": "^5.2.0",
     "eslint-plugin-prettier": "^5.0.1"
   },
   "dependencies": {
-    "@docusaurus/theme-common": "^3.5.0",
```

Comment:
Note: I had to use the canary here otherwise it seems the ^ range doesn't resolve to the demo pre-release, leading to a duplicated package, leading to SSG errors due to providers.

This can be reverted once v3.6 is out